site/_layouts/documentation.html

```diff
@@ -107,6 +107,20 @@
           </nav>
         </div>
         <div class="col-lg-9">
+          <a id="gh-edit" style="visibility: hidden;"><i class="fa fa-pencil" aria-hidden="true"></i> Edit</a>
+          <script>
+            var ghDocsBazeURL = 'https://github.com/bazelbuild/bazel/tree/master/site/versions/master/docs/';
+            var editButton = document.getElementById('gh-edit');
+            // if there is an edit button and we are on a doc page but not in the Build Encyclopedia
+            if(editButton
+              && window.location.pathname.split('docs/').length > 1
+              && window.location.pathname.lastIndexOf('/be/') == -1) {
+              var mdFile = window.location.pathname.split('docs/')[1].replace('html', 'md');
```

Comment:
We still have 4 .html files in that:

$ find site/versions -name '*.html'
site/versions/master/docs/bazel-user-manual.html
site/versions/master/docs/build-ref.html
site/versions/master/docs/query.html
site/versions/master/docs/test-encyclopedia.html


You might want to add special cases for those.